quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesAutoBuildDistTest.java

```diff
@@ -39,38 +36,10 @@
 @TestMethodOrder(OrderAnnotation.class)
 public class QuarkusPropertiesAutoBuildDistTest {
 
-    @Test
```

Comment:
I might be missing something but why are we removing so many tests?

Reply:
@vmuzikar 
All these tests are not relevant anymore as they directly touch the `quarkus.log.console.level` prop, which was not exposed before. In this case, we map to the Qks property from our KC props `--log-console-level`. Even when the Qks prop is set, it's overridden by our property mapper - by a specified value or default value.

"They're testing setting Qks props, should we replace them with sth else?" - IMO, we do not need so many tests around these configs as even before, they did not cover different behavior than the ones that stay (hibernate and datasources).

So, I'd say it's ok to have it like this.
Is it ok?

Reply:
As far as I can tell, the removed tests were not focused on `quarkus.log.console.level` but on other things like if reaug acts as it's supposed to (`reAugOnFirstRun`, `testQuarkusRuntimePropDoesNotTriggerReAug`, `testNoReAugAfterChangingRuntimeProperty`, ...).

Reply:
@vmuzikar You're right, I'll revisit that.

Reply:
Fixed.